mobify/progressive-web-scaffold
# Iframe bridge changes and example

(brief description / PR title)

 **JIRA**: [WEB-989](https://mobify.atlassian.net/browse/WEB-989)
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/388

## Changes
- Build the `content-api.js` script as part of webpack config
- Add example of manual navigation of the iframe, via redux action
- Modify `loader.js` with new case for loading within child frame

## How to test-drive this PR
- Ensure you've followed directions in https://github.com/mobify/progressive-web-sdk/pull/388
- `npm link progressive-web-sdk`
- `npm run dev`
